### PR TITLE
fix PodDisruptionBudget deprecation warning

### PR DESCRIPTION
**Pull request by danieldevlewis**

### CODE REVIEW

File: src/image-resizer/templates/pdb.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {{- if .Values.podDisruptionBudget.enabled }}
 ---
-apiVersion: policy/v1beta1
+apiVersion: policy/v1
```
Comment (michelesr) [inline src/image-resizer/templates/pdb.yaml:3]: have you checked the new api spec is compatible in terms of fields? do we need to change anything?

**Review by michelesr — CHANGES_REQUESTED** [review]
you need to update the chartVersion in Chart.yaml, and publish a new chart:

```
helm package src/image-resizer
```

Then move the generated archive into charts, and update the index with:

```
helm repo index .
```

Then `git add` both the new package and the new index and commit

File: src/image-resizer/templates/pdb.yaml (hunk 1 above)
Comment (danieldevlewis) [reply src/image-resizer/templates/pdb.yaml:3]: It says an empty `spec.selector` will now select all pods.   But we have a selector.  So I think it is fine.

https://kubernetes.io/docs/reference/using-api/deprecation-guide/#poddisruptionbudget-v125

**Comment by danieldevlewis** [comment]
👍 done.  I knew I was missing something

**Comment by danieldevlewis** [comment]
I presume that production is at least on v1.21, or this would be a bad update.

**Review by michelesr — CHANGES_REQUESTED** [review]
You haven't updated the chart version, is still the old one (0.2.2), also the package goes into the charts folder, not in the repo root

**Comment by danieldevlewis** [comment]
Ah, still learning, OK, will sort it tomorrow

**Comment by zhelyan** [comment]
> Ah, still learning, OK, will sort it tomorrow

it's our fault, there's no reason for this to be a manual process - https://citizensadvice.atlassian.net/browse/OPS-4783

**Comment by michelesr** [comment]
@mrdaniellewis I've amended your commit, I think it's okay now.

**Review by michelesr — APPROVED** [review]

**Review by zhelyan — APPROVED** [review]

**Pull request merged**